app/src/main/java/com/example/keybindhelper/ui/keybind/KeybindFragment.kt

```diff
@@ -15,81 +15,98 @@ import com.example.keybindhelper.R
 import com.example.keybindhelper.RecyclerViewAdapters.GroupAdapter
 import com.example.keybindhelper.dao.CurrentProjectManager
 
-
 class KeybindFragment : Fragment() {
-
-
-
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
-
-        val root: View = LayoutInflater.from(this.context).inflate(R.layout.fragment_keybind,container,false)//binding.root
-
-        val mainActivity=activity as MainActivity;
-        if(mainActivity.Menu==null)
-            mainActivity.onMenuInit=(object:MainActivity.MenuInitialized{
+        // Alex code review: https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
+        val root: View = LayoutInflater
+            .from(this.context)
+            .inflate(R.layout.fragment_keybind,container,false) //binding.root
+        val mainActivity = activity as MainActivity
+
+        if(mainActivity.Menu == null)
+            mainActivity.onMenuInit = (object:MainActivity.MenuInitialized {
                 override fun menuHasInitialized() {
-                    initMenu(mainActivity,root);
+                    initMenu(mainActivity, root)
                 }
             })
         else
-            initMenu(mainActivity,root);
+            initMenu(mainActivity,root)
 
-        if(CurrentProjectManager.CurrentProject==null || CurrentProjectManager.CurrentProject.Groups==null) {
+        if(CurrentProjectManager.CurrentProject == null || CurrentProjectManager.CurrentProject.Groups == null) {
             CurrentProjectManager.isProjectLoaded.observe(viewLifecycleOwner) {
                 println("LOADED")
-                loadRecycleView(root);
+                loadRecycleView(root)
             }
         }else{
             println("already loaded")
-            loadRecycleView(root);
+            loadRecycleView(root)
         }
-
-
         return root
     }
 
-    private fun initMenu(mainActivity: MainActivity,view: View) {
-        mainActivity.setAppBarTitle(CurrentProjectManager.CurrentProject.name.value!!);
+    private fun initMenu(mainActivity: MainActivity, view: View) {
+        mainActivity.setAppBarTitle(CurrentProjectManager.CurrentProject.name.value!!)
```

Comment:
While you're in here, one thing I would change is the force unwrap (!!) to a safe non null call with ?.let {}